src/shogun/base/SGObject.h

```diff
@@ -353,6 +494,30 @@ class CSGObject
 	void unset_global_objects();
 	void init();
 
+	/** Checks if object has a parameter identified by a BaseTag.
```

Comment:
What does that mean? Only checks for name and not type?
Can you somehow tell the devs when they would use this?